### DISCUSSION TITLE
some problems

### DISCUSSION THREAD

**Discussion by yosf33** [hieuwu/android-groceries-store#170] [General]
when i tried the source code on my mobile i faced many problem 

1- the photos in the home screen does not download
2-when i searched about apple i didn't receive any action (better i take a message item not found )
3-the scale type of the image they are  incompatable

![WhatsApp Image 2023-05-17 at 9 55 19 PM](https://github.com/hieuwu/android-groceries-store/assets/82787596/b2259465-14d8-4983-a1fa-e40664b9f07c)
![WhatsApp Image 2023-05-17 at 9 55 20 PM](https://github.com/hieuwu/android-groceries-store/assets/82787596/612ee062-b789-4574-b54f-109788d489dd)
![WhatsApp Image 2023-05-17 at 9 55 21 PM](https://github.com/hieuwu/android-groceries-store/assets/82787596/e9a0ca4a-5a2d-4b4d-9fe8-853c27ea7547)

**Comment by hieuwu** [comment]
As looked in your screenshot, looks like when open the app, the data is not retrieved yet. Could you please wait anh bit at onoboarding screen before Get started. (I will improve this)
